src/api/index.js

```diff
@@ -276,6 +276,70 @@ const postNewDrawingSub = async (body, subId) => {
   }
 };
 
+/* 
+***** Uploading to S3 *****
+This function gets an authorized temp url from the BE.
+Then gets the returning data along with the signed request and url 
+Then it preps the upload with the headers and uploads the signed request with the file and returns the url
+After this the postSubmissionPage component will submit the submission's page 
+*/
+const postNewUpload = async file => {
+  try {
+    return apiAuthPost(
+      '/submission/s3',
+      {
+        fileName: file.name.split('.')[0],
+        fileType: file.name.split('.')[1],
+      },
+      getAuthHeader()
+    )
+      .then(res => {
+        let returnData = res.data;
+        let signedRequest = returnData.signedRequest;
+        let url = returnData.url;
+        let options = {
+          headers: {
+            'Content-Type': file.name.split('.')[1],
+          },
+        };
+        return axios
+          .put(signedRequest, file, options)
+          .then(result => {
+            console.log('Response from s3', result);
```

Comment:
If it's not an error, then let's remove console.log

Reply:
got it, removed console.log from line 308